From b6716256659c7cf397320804b200c2afe7a8a622 Mon Sep 17 00:00:00 2001
From: David Greaves <david@dgreaves.com>
Date: Mon, 29 Apr 2019 19:09:47 +0100
Subject: [PATCH 3/3] ZipCheck: Ignore any Exception here

A variety of errors can occur here.
IOError for zero length files, zlib.error for malformed files and
other zipfile errors when opening a file which is not actually a zip archive

Signed-off-by: David Greaves <david.greaves@jolla.com>
---
 ZipCheck.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ZipCheck.py b/ZipCheck.py
index 4da6b06..9b0f55b 100644
--- a/ZipCheck.py
+++ b/ZipCheck.py
@@ -43,7 +43,7 @@ class ZipCheck(AbstractCheck.AbstractCheck):
                     badcrc = z.testzip()
                     if badcrc:
                         printError(pkg, 'bad-crc-in-zip', badcrc, fname)
-                except (RuntimeError, zipfile.error):
+                except (Exception):
                     printWarning(pkg, 'unable-to-read-zip', '%s: %s' %
                                  (fname, sys.exc_info()[1]))
                 else:
-- 
2.11.0

